core/app-framework/app-native-shared/attr_container.c

```diff
@@ -617,18 +654,36 @@ attr_container_get_as_int(const attr_container_t *attr_cont, const char *key)
     TEMPLATE_ATTR_BUF_TO_VALUE(attr_cont, key, i);
 }
 
+uint32_t
+attr_container_get_as_uint32(const attr_container_t *attr_cont, const char *key)
+{
+    TEMPLATE_ATTR_BUF_TO_VALUE(attr_cont, key, e);
```

Comment:
Using macro increases the code size, how about:
```C
    return (uint32_t)attr_container_get_as_int32(attr_cont, key);
```

Reply:
Thanks, I got it.